src/components/productsList/_productsList.scss

```diff
@@ -0,0 +1,121 @@
+$media-tablet: 768px;
+$media-desktop: 1024px;
+$padding-mobile: 20px;
+$padding-desktop: 40px;
+
+.products-list {
+  display: flex;
+  flex-direction: column;
+  margin: 40px 10px 10px;
+
+  * {
+    box-sizing: border-box;
```

Comment:
This should be CSS wide, please move this to a generic SASS file (e.g. `base.scss`